src/wagtail_personalisation/templates/modeladmin/wagtail_personalisation/segment/summary.html

```diff
@@ -0,0 +1,3 @@
+<li class="icon icon-fa-snowflake-o">
```

Comment:
Question: Why is this new?

Reply:
SiteSummaryPanel is a subclass of Component. The Component class no longer has a `render` method. It's actually a change first implemented in v2.15: https://docs.wagtail.org/en/stable/releases/2.15.html#admin-homepage-panels-summary-items-and-action-menu-items-now-use-components

This is were the change was made: https://github.com/wagtail/wagtail-personalisation/pull/240/commits/241af6cd57bfe46e8edadaf225db778338f73e1a#diff-c4a6309c46bb6f60b100be17b7b319c4e96fe47ff4d849ba095e39f25e06b3eaL249 and the template was added for that reason. Should have been in the same commit 😭 sorry.

Do let me know if I've described it right 😄 Thanks

Reply:
Ah okay. Makes more sense.